### PR TITLE
Quick Bar Updates

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

On First Load, Suggestions are shown based on previous clicks
CleanShot 2022-04-15 at 23 07 55

Typing limits the categories and entries. 
CleanShot 2022-04-15 at 23 08 15

All categories are shown on the single search
CleanShot 2022-04-15 at 23 09 18

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to an issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by spacegaier** [comment]
So 'c' and 'e' will then basically open the same quick bar, since it will now be a combined one?

**Comment by zsarnett** [comment]
Thats correct

File: src/dialogs/quick-bar/ha-quick-bar.ts [hunk 1]
```diff
@@ -2,147 +2,161 @@ import "@lit-labs/virtualizer";
...
-      this._commandItems,
-      this._entityItems,
-      this._filter
-    );
+    let sectionCount = 0;
+
+    if (this._items && this._filter && this._filter !== "") {
+      const newFilteredItems: QuickBarItem[] = [];
+      this._items.forEach((arr) => {
+        const items = this._filterItems(arr, this._filter).slice(0, 3);
```
Comment (bramkragten) [inline src/dialogs/quick-bar/ha-quick-bar.ts:145]: Why do we show only 3 items per section? Searching an entity by domain will be very hard now?

**Comment by matthiasdebaat** [comment]
We will pick this up later

**Pull request closed without merging**